colors/everforest.vim

```diff
@@ -139,9 +139,11 @@ call everforest#highlight('PmenuThumb', s:palette.none, s:palette.grey0)
 if s:configuration.float_style ==# 'dim'
   call everforest#highlight('NormalFloat', s:palette.fg, s:palette.bg_dim)
   call everforest#highlight('FloatBorder', s:palette.grey1, s:palette.bg_dim)
+  call everforest#highlight('FloatTitle', s:palette.fg, s:palette.bg_dim, 'bold')
```

Comment:
It should be sufficient to set this highlight group once, with a background value of `s:palette.none`, this way the background color is inherited from NormalFloat.

Reply:
Edit: my bad, the title is located on the border, not inside the window.